src/utils/time.ts

```diff
@@ -1,9 +1,18 @@
-import { format, formatDuration, formatRelative } from "date-fns";
+import {
+  differenceInSeconds,
+  format,
+  formatDuration,
+  formatRelative,
+  fromUnixTime,
+  getUnixTime,
+  startOfDay,
+} from "date-fns";
 
 import { VenueEvent } from "types/venues";
 
 export const ONE_SECOND_IN_MILLISECONDS = 1000;
 export const ONE_MINUTE_IN_SECONDS = 60;
+export const ONE_HOUR_IN_MINUTES = 60;
 export const ONE_HOUR_IN_SECONDS = ONE_MINUTE_IN_SECONDS * 60;
 export const ONE_DAY_IN_SECONDS = ONE_HOUR_IN_SECONDS * 24;
```

Comment:
I searched for all the constants. Each of them is used somewhere. We might add a `@deprecated` / `@dept` comment to them encouraging developers to use date-fns functions instead?

Reply:
(FYI: `@debt` not `@dept`)

Sounds like a good idea, let's add the `/**` form of JSdoc comments + `@deprecated` to the constants themselves. And then add `@debt` comments saying to refactor to use `utils/time` and/or `date-fns` at all the areas in the code that are currently using the consts.

Reply:
fixed cases where I got an idea about how to fix that fast and added `@debt` comments in other places: https://github.com/sparkletown/sparkle/pull/1302/commits/665b77b03c0d0cf316ef4813f7f2f6ddd4ffd785

btw, failed to spell `@debt` again in the commit message. :woman_facepalming:

Reply:
in other cases constants are used more like "amount of time".
As an idea / suggestion we could have one more util: time converter

```js
time(5).secondsToMilliseconds()
time(10).hoursToMilliseconds()
```

but it can overcomplicate I guess.

Reply:
> As an idea / suggestion we could have one more util: time converter

Is this using an existing dependency/lib we have? Or were you thinking we could write a new one/similar?

Reply:
I was thinking about creating our own new converter. Don't know if there any existent one.
But yeah, agree, that approach seem to be not "worth enough".

Reply:
Yeah, let's not for now and see if it comes up as an idea/annoyance again in future.